src/i18n/RuleEngine.js

```diff
@@ -509,27 +509,22 @@ export default {
   },
   clientPoolsize: {
-    zh: '客户端池大小',
-    en: 'MQTT Client Pool Size',
+    zh: '连接池大小',
+    en: 'Connection Pool Size',
   },
   egressPoolSizeDesc: {
-    zh: `MQTT 客户端池的大小，这些客户端将会向远程代理发布消息。<br/>
-    每个 MQTT 客户端都将被分配一个 \`clientid\`，形式为 \`\${'{'}clientid_prefix{'}'}:\${'{'}bridge_name{'}'}:egress:\${'{'}node{'}'}:\${'{'}n{'}'}\`。其中，\`n\` 是池中客户端的编号。`,
-    en: `Size of an MQTT client pool. MQTT clients in the pool publish messages to the remote broker.<br/>
-Each MQTT client will be assigned \`clientid\` in the form of \`\${'{'}clientid_prefix{'}'}:\${'{'}bridge_name{'}'}:egress:\${'{'}node{'}'}:\${'{'}n{'}'}\`
-where \`n\` is the number of a client inside the pool.`,
+    zh: `用于出口配置的 MQTT 客户端连接池大小。<br/>
+    连接池中每个 MQTT 客户端都将被分配一个唯一的 \`clientid\` 以确保避免重复或冲突，格式为 \`\${clientid_prefix}:\${bridge_name}:egress:\${node}:\${n}\`，其中 \`n\` 是连接池中客户端的编号。`,
+    en: `The size of the MQTT client connection pool for egress. <br/>
+Each MQTT client in the connection pool will be assigned a unique clientid to ensure avoidance of duplication or conflicts. The format of the clientid is \`\${clientid_prefix}:\${bridge_name}:egress:\${node}:\${n}\`, where \`n\` represents the number of the client in the connection pool.`,
   },
   ingressPoolSizeDesc: {
-    zh: `MQTT 客户端池的大小，这些客户端将从远程代理接收消息。<br/>
-仅当 \`remote.topic\` 为共享订阅主题或主题过滤器（例如 \`$share/name1/topic1\` 或 \`$share/name2/topic2/#\`）时，才会尊重此值，否则将仅使用单个 MQTT 客户端。
-每个 MQTT 客户端都将被分配一个 \`clientid\`，形式为 \`\${'{'}clientid_prefix{'}'}:\${'{'}bridge_name{'}'}:ingress:\${'{'}node{'}'}:\${'{'}n{'}'}\`。其中， \`n\` 是池中客户端的编号。
-注意：当 EMQX 处于集群状态时，非共享订阅将无法良好地工作。`,
-    en: `Size of an MQTT client pool. MQTT clients in the pool ingest messages from the remote broker.<br/>
-This value will be respected only if 'remote.topic' is a shared subscription topic or topic-filter
-(for example \`$share/name1/topic1\` or \`$share/name2/topic2/#\`). Otherwise only a single MQTT client will be used.
-Each MQTT client will be assigned 'clientid' of the form '\${'{'}clientid_prefix{'}'}:\${'{'}bridge_name{'}'}:ingress:\${'{'}node{'}'}:\${'{'}n{'}'}'
-where 'n' is the number of a client inside the pool.
-NOTE: Non-shared subscription will not work well when EMQX is clustered.`,
+    zh: `用于入口配置的 MQTT 客户端连接池大小。<br/>
+仅当 远程主题（\`remote.topic\`） 使用了共享订阅（例如 \`$share/my-group/topic1\`）时才会启用连接池。
+连接池中每个 MQTT 客户端都将被分配一个唯一的 \`clientid\` 以确保避免重复或冲突，格式为 \`\${clientid_prefix}:\${bridge_name}:ingress:\${node}:\${n}\`，其中 \`n\` 是连接池中客户端的编号。`,
+    en: `The size of the MQTT client connection pool for egress. <br/>
+The connection pool is enabled only when \`remote.topic\` is using shared subscriptions (e.g., \`$share/my-group/topic1\`). <br/>
+Each MQTT client in the connection pool will be assigned a unique clientid to ensure avoidance of duplication or conflicts. The format of the clientid is \`\${clientid_prefix}:\${bridge_name}:ingress:\${node}:\${n}\`, where \`n\` represents the number of the client in the connection pool.`,
```

Comment:
```suggestion
Each client in the MQTT connection pool is allocated a unique client ID to prevent duplication or conflicts. The client ID follows the format: \`\${clientid_prefix}:\${bridge_name}:ingress:\${node}:\${n}\`, where \`n\` represents the client's number in the connection pool.`,
```